lib/MerkleMultiProof.sol

```diff
@@ -8,7 +8,7 @@ struct Node {
 
 library MerkleMultiProof {
     function verifyProof(bytes32 root, Node[][] memory proof, Node[] memory leaves)
-        public
+        external
```

Comment:
change this to internal. To make the tests pass create a `contract` instead that uses the library